### DISCUSSION TITLE
nextjs 13 with layout

### DISCUSSION THREAD

**Discussion by Manubi** [shuding/nextra#1056] [Q&A]
Hi,
is there a possibility that I can wrap my `01_blog_heyho.mdx` with the `layout.tsx` component which is available in nextjs13?
![CleanShot 2022-12-09 at 19 00 46](https://user-images.githubusercontent.com/20935138/206764749-6acbde9e-0830-4e09-9055-88aae0b3fc85.png)

**Comment by shuding** [comment]
Unfortunately I don't think that's possible at the moment
